Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealityDeviceManager.cs

```diff
@@ -599,9 +652,30 @@ public override void Disable()
             }
         }
 
-        #endregion IMixedRealityDeviceManager Interface
+#endregion IMixedRealityDeviceManager Interface
+
+                        #region Controller Utilities
```

Comment:
There's some inconsistent indentation changes in this class, specifically around the regions